src/test/java/sql/jpa/JpaTest.java

```diff
@@ -42,15 +40,9 @@
     
     @BeforeClass
     public static void setUpClass() {
-        BasicConfigurator.configure();
-        Logger.getRootLogger().setLevel(Level.INFO);//JPA is very verbose...
+        System.setProperty("org.jboss.logging.provider", "slf4j"); // The JBoss logging discovery process ius not so reliable...
```

Comment:
Mh, if you run the tests without that property hibernate use the JDK logger, not SLF4J (I am going to double check)